examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,507 @@
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct CreateChatThreadResult : Swift.Codable {
+  public let chatThread: AzureCommunicationChat.ChatThreadProperties?
+  public let invalidParticipants: [AzureCommunicationChat.ChatError]?
+  public init(chatThread: AzureCommunicationChat.ChatThreadProperties? = nil, invalidParticipants: [AzureCommunicationChat.ChatError]? = nil)
+  public func encode(to encoder: Swift.Encoder) throws
+  public init(from decoder: Swift.Decoder) throws
+}
+public struct SendChatMessageOptions : AzureCore.RequestOptions {
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct SendChatMessageResult : Swift.Codable {
+  public let id: Swift.String
+  public init(id: Swift.String)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct CommunicationUserIdentifierModel : Swift.Codable {
+  public let id: Swift.String
+  public init(id: Swift.String)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct AddChatParticipantsOptions : AzureCore.RequestOptions {
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct CommunicationIdentifierModel : Swift.Codable {
+  public let rawId: Swift.String?
+  public let communicationUser: AzureCommunicationChat.CommunicationUserIdentifierModel?
+  public let phoneNumber: AzureCommunicationChat.PhoneNumberIdentifierModel?
+  public let microsoftTeamsUser: AzureCommunicationChat.MicrosoftTeamsUserIdentifierModel?
+  public init(rawId: Swift.String? = nil, communicationUser: AzureCommunicationChat.CommunicationUserIdentifierModel? = nil, phoneNumber: AzureCommunicationChat.PhoneNumberIdentifierModel? = nil, microsoftTeamsUser: AzureCommunicationChat.MicrosoftTeamsUserIdentifierModel? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct AddChatParticipantsRequest : Swift.Codable {
+  public let participants: [AzureCommunicationChat.ChatParticipant]
+  public init(participants: [AzureCommunicationChat.ChatParticipant])
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
```

Comment:
Why is this public? Your public method takes the `[ChatParticipant]` collection directly, so I would think this should not be part of the public API.